### ISSUE TITLE
lineClamp not supported anymore on <Title />?

### ISSUE THREAD

**Issue by atumas-bananamilk** [mantinedev/mantine#5321]
### What package has an issue?

@mantine/core

### Describe the bug

v7 doesn't support `lineClamp` for `<Title />` anymore?

![Screenshot 2023-11-24 at 18 20 08](https://github.com/mantinedev/mantine/assets/22228431/4f3b108d-3584-4b1f-8f6a-2eb8c2ef7094)

### What version of @mantine/* packages do you have in package.json? (Note that all @mantine/* packages must have the same version in order to work correctly)

^7.2.2

### Do you know how to fix the issue

None

### Do you want to send a pull request with a fix?

None

**Comment by jt561** [comment]
Yes. The docs states that it's been removed from title. I've had to result to just using Text

**Comment by atumas-bananamilk** [comment]
I guess will have to completely migrate to `<Text />` from now on.
Makes no sense but ok.

Migrations:
```
    <Title order={3} lineClamp={1}>Nice one</Title>
    <Text fz={22} fw={700} lineClamp={1}>Nice one</Text> // identical to Title order = 3

    <Title order={2} lineClamp={1}>Nice one</Title>
    <Text fz={26} fw={700} lineClamp={1}>Nice one</Text> // identical to Title order = 2

    <Title order={1} lineClamp={1}>Nice one</Title>
    <Text fz={34} fw={700} lineClamp={1}>Nice one</Text> // identical to Title order = 1
```

**Comment by jt561** [comment]
Yeah honestly it's a shame but there's nothing else we can do. I guess you can also add `component="h1"` too which should add the h1 tag.

**Comment by rtivital** [comment]
Fixed in [7.3.1](https://github.com/mantinedev/mantine/releases/tag/7.3.1)
